Unexported types

Hey,

After latest release we couldn't build our project anymore with this error:
```
error TS4029: Public property 'rateLimiter' of exported class has or is using name 'GraphQLRateLimitDirectiveArgs' from external module "/codebuild/output/src845456456/src/github.com/deepcrawl/canary/node_modules/graphql-rate-limit/dist/index" but cannot be named.
```

we are using option 3 to setup rate limited, we were able to patch existing release to export types.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/graphql-rate-limit/dist/index.d.ts b/node_modules/graphql-rate-limit/dist/index.d.ts
index 1af0ba1..656aa78 100644
--- a/node_modules/graphql-rate-limit/dist/index.d.ts
+++ b/node_modules/graphql-rate-limit/dist/index.d.ts
@@ -36,7 +36,7 @@ interface Identity {
  *
  * myField(id: String): Field @rateLimit(message: "Stop!", window: 100000, max: 10, identityArgs: ["id"])
  */
-interface GraphQLRateLimitDirectiveArgs {
+export interface GraphQLRateLimitDirectiveArgs {
     /**
      * Error message used when/if the RateLimit error is thrown
      */
```

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,5 +1,6 @@
 import { createRateLimitDirective } from './lib/field-directive';
 
+export * from './lib/types';
 export * from './lib/field-directive';
 export * from './lib/store';
 export * from './lib/in-memory-store';

